src/picongpu/include/plugins/kernel/CopySpecies.kernel

```diff
@@ -63,7 +63,7 @@ struct ConcatListOfFrames
      * @param mapper mapper which describe the area were particles are copied from
      */
     template<class T_DestFrame, class T_SrcBox, class T_Filter, class T_Space, class T_Mapping>
-    void operator()(int& counter, T_DestFrame destFrame, T_SrcBox srcBox, T_Filter filter, T_Space particleOffset, T_Mapping mapper)
+    void operator()(int& counter, T_DestFrame destFrame, T_SrcBox srcBox, T_Filter filterWindowDomain, T_Space particleOffset, T_Mapping mapper)
```

Comment:
why is `counter` given by ref, do you return it? (not documented in doxygen as output parameter)